From 70ff9d15f67dc88b6d81f794e8bb35820c23dfc2 Mon Sep 17 00:00:00 2001
From: Robert Mader <robert.mader@collabora.com>
Date: Tue, 15 Feb 2022 21:31:06 +0100
Subject: [PATCH 13/18] wayland/dma-buf: Fix typos in struct name

Fixes 7acecb1c7233dad0b9b314f59121882a518901d7

Part-of: <https://gitlab.gnome.org/GNOME/mutter/-/merge_requests/2297>
(cherry picked from commit 8f91d831ee20b9c73036561ee1b57bf9f5efef65)
---
 src/wayland/meta-wayland-dma-buf.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/wayland/meta-wayland-dma-buf.c b/src/wayland/meta-wayland-dma-buf.c
index 4e21632159..f28070da80 100644
--- a/src/wayland/meta-wayland-dma-buf.c
+++ b/src/wayland/meta-wayland-dma-buf.c
@@ -1281,7 +1281,7 @@ add_fallback:
  *
  * [ 32 bit format  ][ 32 bit padding ][          64 bit modifier         ]
  */
-typedef struct _MetaMetaWaylanDmaBdufFormatEntry
+typedef struct _MetaWaylandDmaBufFormatEntry
 {
   uint32_t drm_format;
   uint32_t unused_padding;
-- 
2.51.1

